pkg/builders/default_storage_strategy.go

```diff
@@ -64,6 +64,9 @@ func (DefaultStorageStrategy) Build(builder StorageBuilder, store *StorageWrappe
 	store.CreateStrategy = builder
 	store.UpdateStrategy = builder
 	store.DeleteStrategy = builder
+	if table, ok := builder.(rest.TableConvertor); ok {
```

Comment:
thx for contributing this. while this is hackier than i thought. note that server-side printing should only work w/ unversioned/hub type in the storage. extend this feature base on the unversioned builder plz.